.gitmodules

```diff
@@ -1,12 +1,12 @@
 [submodule "phoenix-rtos-kernel"]
 	path = phoenix-rtos-kernel
-	url = https://github.com/phoenix-rtos/phoenix-rtos-kernel.git
+	url = https://github.com/jsarzy/phoenix-rtos-kernel.git
```

Comment:
this should not be a part of this PR, it would need to be fixed before merging.

You may update the PR after we would merge respective PRs in the submodules affected (so: I'm leaving this comment not to merge this PR by accident)